frontend/src/components/node/Details.tsx

```diff
@@ -119,6 +120,10 @@ export default function NodeDetails() {
     }, 1000);
   }
 
+  function handleDrainDialog() {
```

Comment:
Nit: This could have a more descriptive name, like `toggleDrainDialogVisibility`. The rest looks like.

Reply:
done!